File: week6/mario/mario.py
from cs50 import get_int

def piramide (height):
    for x in range(height):
        for y in range(height - x - 1):
            print(f" ", end="")
        for y in range(x+1):
            print(f"#",end="")
        print(f"  ",end="")
        for y in range(x+1):
            print(f"#",end="")
        print(f"\n",end="")

h = 0
while h<1 or h>8:
    h = get_int("Height: ")
piramide(h)

File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void piramide(int height){
    for(int i = 0;i<height;i++){
        for(int j = 0;j<height - i -1;j++){
            printf(" ");
        }
        for (int j = 0; j<=i;j++){
            printf("#");
        }
        printf("  ");
        for (int j = 0; j<=i;j++){
            printf("#");
        }
        printf("\n");
    }
}

int h;
int main(void){
    do {
        h = get_int("Height: ");
    }while(h < 1 || h>8);
    piramide(h);
}




/*

      ##  ##
     ###  ###
    ####  ####
   #####  #####
  ######  ######
 #######  #######
########  ########
*/
